modules/microshift-gathering-sos-report.adoc

```diff
@@ -18,7 +18,7 @@
 +
 [source,terminal]
 ----
-$ sos report --batch --clean --all-logs --profile microshift
+$ sos report --batch --all-logs --profile microshift
```

Comment:
Should this documentation be using `/usr/bin/microshift-sos-report` to collect SOS logs  ?

Reply:
@copejon WDYT for this?